libs/src/api/rewards.js

```diff
@@ -199,12 +201,12 @@ class Rewards extends Base {
         throw new Error(evaluateErrorCode || evaluateError)
       }
 
-      return { success: true, error: null, phase: null, nodesToReselect: null }
+      return { success: true, error: null, errorCode: errorCode, phase: null, nodesToReselect: null }
```

Comment:
style nit: you should never need to do something like `{ errorCode: errorCode }`, you can just do `{ errorCode }` -this is called punning, strangely enough ha https://twitter.com/amytongwu/status/1556993991329062919

also I think the type may be wrong here? In the below branch we return aaoErrorCode, here we just return errorCode (which we should set to null to be consistent)

Reply:
Went through and explicitly returned null aaoErrorCode in all return statements.